googletest/src/gtest.cc

```diff
@@ -3156,7 +3156,7 @@ static void PrintTestPartResult(const TestPartResult& test_part_result) {
 
 // class PrettyUnitTestResultPrinter
 #if GTEST_OS_WINDOWS && !GTEST_OS_WINDOWS_MOBILE && !GTEST_OS_WINDOWS_PHONE && \
```

Comment:
It seems that google test is using stuff like
```cpp
#if defined(GTEST_OS_WINDOWS)
```

![image](https://user-images.githubusercontent.com/79500078/223611600-5a091e2a-ef97-4346-b685-db0f79b48366.png)

Is there a big difference between the old `#if GTEST_OS_WINDOWS` here?